Sources/SwiftSyntax/RawSyntax.swift

```diff
@@ -1113,7 +1113,7 @@ final class RawSyntax: ManagedBuffer<RawSyntaxBase, RawSyntaxDataElement> {
     }
   }
 
-  func withLeadingTrivia(_ leadingTrivia: Trivia) -> RawSyntax {
+  func withLeadingTrivia(_ leadingTrivia: Trivia) -> RawSyntax? {
```

Comment:
Could you add documentation here what returning `nil` means? I’d suggest
```swift
/// Replaces the leading trivia of the first token in this syntax tree by `leadingTrivia`.
/// If the syntax tree did not contain a token and thus no trivia could be attached to it, `nil` is returned.
```

Similar below.

Reply:
Added! 
Also added parameter documentation